src/hotspot/share/runtime/cpuTimeCounters.hpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2002, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This is probably on me, but this 2002 is wrong:
```suggestion
 * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```